src/lwc/geFormRenderer/geFormRenderer.html

```diff
@@ -92,19 +92,6 @@
                             </ul>
                         </c-util-page-level-message>
                     </template>
-                    <template if:true={showElevateTransactionWarning}>
```

Comment:
Should this Elevate Transaction Warning block be removed?

Reply:
Good catch! @sokosaurus. It shouldn't be removed.

Reply:
Fixed!

Reply:
Perhaps something is incorrectly cached on my side of things, but it looks like this block of code is still being removed from the PR.

Reply:
@voduyemi It appears this code block is still not in the branch - could you make sure to add this back?